canonical-web-and-design/ubuntu.com
# Remove 2 iot boards Joule and Artik

## Done

- Remove the Samsung Artik and the Intel Joule from the site.
- Updated copy docs and archived the install ones

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - https://0.0.0.0:8001/#download
    - https://0.0.0.0:8001/download/iot/index
    - https://0.0.0.0:8001/download/intel-nuc
    - https://0.0.0.0:8001/download/intel-joule-desktop
    - https://0.0.0.0:8001/download/intel-joule
    - https://0.0.0.0:8001/internet-of-things/index
    - https://0.0.0.0:8001/download/samsung-artik-5-10-server
    - https://0.0.0.0:8001/download/samsung-artik-5-10
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure you don't see the Joule or Artik anywhere


## Issue / Card

Fixes #6563
